### ISSUE TITLE
False negative for .is_valid()

### ISSUE THREAD

**Issue by Mmdixon** [rspeer/langcodes#43]
```
>>> Language.get("cu-Cyrs").is_valid()
False
```
Expected to return `True`. Fails on the `Cyrs` script tag

Context ([wikipedia](https://cu.wikipedia.org/wiki/%D0%93%D0%BB%D0%B0%D0%B2%D1%8C%D0%BD%D0%B0_%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0)):
`cu`: Old Church Slavonic
`Cyrs`: Cyrillic (Old Church Slavonic variant)
Both these tags are part of https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

**Comment by rspeer** [comment]
You're right -- langcodes can even describe it:

```
>>> langcodes.get('cu-Cyrs').describe()
{'language': 'Church Slavic', 'script': 'Old Church Slavonic Cyrillic'}
```

I'll look into whether the "validity" file I got from CLDR was wrong, or whether I parsed it wrong.

**Comment by rspeer** [comment]
A few scripts are just plain missing from CLDR's validity.xml, it seems.

I didn't want to mess with the CLDR part in this release. Is it okay if I save this for a version based on CLDR v40 when it's released?

**Comment by rspeer** [comment]
Fixed in version 3.3.
